subsys/net/lib/coap/coap.c

```diff
@@ -1750,7 +1750,8 @@ size_t coap_pendings_count(struct coap_pending *pendings, size_t len)
 }
```

Comment:
In the test commit, please do not mix code from `tests/net/...` and `subsys/net/....` in the same commit, but place the changes to `subsys/net/lib/coap/coap.c` to the first commit.
Also please change the test commit subject to `tests: net: coap: ....`